x-pack/packages/kbn-cloud-security-posture/public/src/hooks/use_navigate_findings.ts

```diff
@@ -39,3 +39,17 @@ export const useNavigateFindings = () => {
 
 export const useNavigateVulnerabilities = () =>
   useNavigate(findingsNavigation.vulnerabilities.path);
+
+export const useNavigateNativeVulnerabilities = () => {
```

Comment:
In the long term we will probably **have** to pass the filter from the findings on click, which makes me think it might be better to just add the filter prop to the original `useNavigateVulnerabilities` hook and pass the `data_stream` filter to it, since the page already filtered for our data stream anyway.

though, If its technically complex we can change this later

Reply:
yep, let's change it later. Main reason, to make smaller changes as possible for 8.16 after the ff